server/index.js

```diff
@@ -1,16 +1,29 @@
 const express = require('express');
+const path = require('path');
 let app = express();
-let { retrieve, seeder } = require('./db/index.js')
+let bodyParser = require ('body-parser');
+let { retrieve } = require('./db/index.js')
+let port = 4000;
 
 app.use(express.static(__dirname + '/../client/dist'));
 
-app.get('/', function (req, res) {
-  console.log('Got request');
-  seeder();
-  res.end();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.get('/buildings/:id', function (req, res) {
+  console.log(`Got request for ${req.params.id}`);
+  let index = path.resolve(__dirname + '/../client/dist/index.html');
+  let options = { id: req.params.id };
+  res.sendFile(index, options);
```

Comment:
What are you achieving here? I interpret this as the server will respond with an HTML template, but I'm not sure how the `options` parameter will be passed through.

Reply:
The intention was to load index.html when someone went to /buildings/id, and to pass that id along, but I am now realizing that this is probably the wrong approach, as you pointed out its not really using the options, and it still fails to load bundle.js. I'll have to do more research for this part